tests/generated/wasi/io/v0.2.0/streams/abi.go

```diff
@@ -0,0 +1,22 @@
+// Code generated by wit-bindgen-go. DO NOT EDIT.
+
+package streams
+
+import (
+	"go.bytecodealliance.org/cm"
+)
+
+// ResultListU8StreamError represents the imported result "#".
+//
+//	result<list<u8>, stream-error>
+type ResultListU8StreamError cm.Result[cm.List[uint8], cm.List[uint8], StreamError]
+
+// ResultU64StreamError represents the imported result "#".
+//
+//	result<u64, stream-error>
+type ResultU64StreamError cm.Result[uint64, uint64, StreamError]
+
+// ResultStreamError represents the imported result "#".
+//
+//	result<_, stream-error>
+type ResultStreamError cm.Result[StreamError, struct{}, StreamError]
```

Comment:
nice this will reduce the burden of spelling out the shape of the Result type everytime you use it.

Reply:
I'm not sure it's better!